### DISCUSSION TITLE
Possible bug when exporting data

### DISCUSSION THREAD

**Discussion by joeytzy** [Budibase/budibase#11606] [Help] [answered]
Hi, so I'm currently using budibase version 2.8.2. So I encounter an issue when trying to export data as csv.

![image](https://github.com/Budibase/budibase/assets/44018390/1b3286e6-0db8-4c3b-9c5c-00458f5c2dc0)

Image above is how I set up the Export Data.

So I try it out in preview and it returns an error asking for one row to be selected.

![image](https://github.com/Budibase/budibase/assets/44018390/c0dff13a-da76-4f2c-82f5-83b9f5aa7940)

Am I missing something or doing something wrong here or is it a bug.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @joeytzy,

You need to make sure that you have a table component with row selection is toggled on. Once you have selected the rows you wish to export and then hit this action it will export all rows that were selected. Converts and downloads them as a CSV. 
<img width="312" alt="Screenshot 2023-08-29 at 08 03 05" src="https://github.com/Budibase/budibase/assets/126772285/acbe0e7e-7d38-45a9-8ff7-4d515bd3a10e">

I hope this makes sense.

**Reply by joeytzy** [reply]
thanks. Worked as intended
